package.json

```diff
@@ -14,26 +14,32 @@
 		"dev": "env-cmd node --enable-source-maps dist/index.js",
 		"deploy:commands": "env-cmd node --enable-source-maps dist/deploy.js",
 		"migrate": "env-cmd ley",
-		"postinstall": "patch-package"
+		"postinstall": "patch-package",
+		"start:dev": "npm run build && npm run dev"
 	},
 	"type": "module",
 	"keywords": [],
 	"author": "iCrawl <icrawltogo@gmail.com>",
 	"license": "AGPL-3.0",
 	"private": true,
 	"dependencies": {
+		"@aero/sanitizer": "^1.2.0",
 		"@discordjs/rest": "^0.5.0",
 		"@naval-base/ms": "^3.1.0",
 		"bree": "^9.1.0",
 		"bufferutil": "^4.0.6",
 		"common-tags": "^1.8.2",
+		"confusables": "^1.1.1",
 		"dayjs": "^1.11.3",
 		"diff": "^5.1.0",
 		"discord.js": "^14.0.0-dev.1656864763-c7a205f",
 		"env-cmd": "^10.1.0",
 		"i18next": "^21.8.11",
 		"i18next-fs-backend": "^1.1.4",
+		"image-hash": "^5.3.1",
 		"ioredis": "^5.1.0",
+		"kleur": "^4.1.5",
+		"leven": "^4.0.0",
```

Comment:
at scale, considering https://www.npmjs.com/package/fastest-levenshtein might be preferable, if we can trust their benchmarks

Reply:
Wow, If the benchmarks are real, this library is like 10x faster